server/server.go

```diff
@@ -296,10 +308,18 @@ func (s *Server) concurrentRequest(ctx context.Context, conf map[string]lookout.
 
 			cs, err := send(ctx, a.Client, settings)
 			if err != nil {
-				aLogger.Errorf(err, "analysis failed")
+				grpcStatus := status.Convert(err)
+				errMessage, ok := logErrorMessages[grpcStatus.Code()]
+				if !ok {
+					errMessage = fmt.Sprintf("code: %s - message: %s", grpcStatus.Code(), grpcStatus.Message())
+				}
+
+				aLogger.Errorf(err, "analysis failed: %s", errMessage)
```

Comment:
aren't we repeat code&message here?

Reply:
Are you talking about errors without a friendly message? In that case the logged message is something like this:

```msg="analysis failed: code: Unknown - message: review error" analyzer=Dummy app=lookoutd error="rpc error: code = Unknown desc = review error"```

so `code` and `message` are repeated in the `error` object of the log context. Maybe something as follows is better?

```
	if err != nil {
		grpcStatus := status.Convert(err)
		errMessage := "analysis failed"
		friendlyMessage, ok := logErrorMessages[grpcStatus.Code()]
		if ok {
			errMessage = fmt.Sprintf("%s: %s", errMessage, friendlyMessage)
		}

		aLogger.Errorf(err, errMessage)
		...
	}
```

Reply:
Yep. It makes more sense for me.

Reply:
Addressed [here](https://github.com/src-d/lookout/pull/442/commits/415404d302d7b4fd1f95eb32b3c58c3517cff31e).